src/popup/router/pages/SecuritySettings.vue

```diff
@@ -97,7 +97,7 @@ export default {
   methods: {
     setBackedUpSeed() {
       this.$store.commit('setBackedUpSeed');
-      this.$router.push({ name: 'account' });
+      this.$router.push({ name: 'settings' });
```

Comment:
The settings route doesn't have `name`. 

So either add it or:
```suggestion
      this.$router.push({ path: '/settings' });
```

Or even go back (seems less coupled to me):

```suggestion
      this.$store.dispatch('router/back');
```

Reply:
The `router/back` seems like the nicer approach to me as well. Thanks!